chevah_build

```diff
@@ -634,9 +634,28 @@ command_test() {
     echo '##### Executing Chevah tests... #####'
     execute $LOCAL_PYTHON_BINARY ${test_file}
     echo '##### Executing tests for included Python modules... #####'
-    # UTF-8 env needed. Details at https://github.com/benhoyt/scandir/issues/78.
-    LC_ALL=en_US.UTF-8 execute $LOCAL_PYTHON_BINARY \
-        ../python-modules/python-scandir-1.5/test/run_tests.py
+    echo 'Testing scandir...'
+    # There are a series of issues with the scandir tests in Linux/Unix.
+    # Reported upstream at https://github.com/benhoyt/scandir/issues/78.
+    case $OS in
+        osx*|macos*|windows*)
```

Comment:
why not execute these tests with `LC_ALL=en_US.UTF-8` ? ... so that we don't have a separate case for this